src/main/service/probate-man-service.ts

```diff
@@ -5,13 +5,24 @@ import * as userReqAuth from "../user/user-request-authorizer";
 export function getProbateManLegacyCase(req, probateManType, id) {
   const url = get("case_data_probate_template_url") + "/probateManTypes/" + probateManType + "/cases/" + id;
   const authorization = req.get(userReqAuth.AUTHORIZATION);
-  return fetch(url, {
-                      headers: {
-                        "Authorization": authorization,
-                        "Content-Type": "application/json",
-                        "ServiceAuthorization": req.headers.ServiceAuthorization,
-                      },
-                      method: "GET",
-                    })
+  const luhn = require("luhn");
+  if (luhn.validate(id)) {
+    return fetch(url, {
+      headers: {
+        "Authorization": authorization,
+        "Content-Type": "application/json",
+        "ServiceAuthorization": req.headers.ServiceAuthorization,
+      },
+      method: "GET",
+    })
     .then((res) => res.json());
-}
+  } else {
+      return ERROR_BAD_REQUEST;
+      }
+    }
+
+export const ERROR_BAD_REQUEST = {
+  error: "Bad Request",
+  message: "Case ID must be a valid luhn number",
+  status: 400,
+};
```

Comment:
The logic here could be fitted into controller layer, perhaps but not necessarily calling a dedicated luhn validation service object wrapping luhn validator library. Then, we could re-use it for the other endpoint as well. This would also be more consistent with our general approach (if not applied everywhere) to validate data to API operations at controller layers.